### PR TITLE
Harigovind/ch44/setup in app notifications

### PR DESCRIPTION
**Pull request by harigovind1998**
Known Issues:
- After you press reset, the button says resume instead of start
- Pressing the notification does not open the app. Will be better implemented when the notification is updated to use default buttons

### CODE REVIEW

File: app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt [hunk 1]
```diff
@@ -56,10 +58,12 @@ class CircuitTimerActivity : AppCompatActivity() {
     private var tone: ToneGenerator = ToneGenerator(AudioManager.STREAM_MUSIC, 100)
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+        super<AppCompatActivity>.onCreate(savedInstanceState)
         setContentView(R.layout.activity_circuit_timer)
         bind = DataBindingUtil.setContentView(this, R.layout.activity_circuit_timer)
 
+        createNotificationChannel()
```
Comment (SVD6) [inline app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt]: Create all the notifications at the start of the app so Main Activity or even Splash Activity

File: app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt [hunk 2]
```diff
@@ -356,4 +361,35 @@ class CircuitTimerActivity : AppCompatActivity() {
             0
         }
     }
+
+    private fun createNotification(time: Float) {
+        val builder =
+            NotificationCompat.Builder(this, getString(R.string.timer_notification_channel_id))
+                .setSmallIcon(R.drawable.ic_notification_logo)
+                .setContentTitle("Notification test")
```
Comment (SVD6) [inline app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt]: Is this needed right now?

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -35,6 +35,21 @@
     <dimen name="huge_element_padding">85dp</dimen>
     <dimen name="xl_huge_element_padding">90dp</dimen>
     <dimen name="massive_element_padding">100dp</dimen>
+    <!-- View margin   -->
```
Comment (SVD6) [inline app/src/main/res/values/dimens.xml]: Wait isn't this what we already have?

File: app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt (hunk 1 above)
Comment (harigovind1998) [reply app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt]: Disabling Timer notification for now. Still creating a notification channel for timer AND stopwatch notification in Main Activity.

File: app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt (hunk 2 above)
Comment (harigovind1998) [reply app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt]: Commented out the part that issues the notification.

File: app/src/main/java/ca/chronofit/chrono/circuit/CircuitTimerActivity.kt [hunk 4]
```diff
@@ -62,10 +72,11 @@ class CircuitTimerActivity : BaseActivity() {
     private var tone: ToneGenerator = ToneGenerator(AudioManager.STREAM_MUSIC, 100)
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+        super<AppCompatActivity>.onCreate(savedInstanceState)
```
Comment (SVD6) [inline app/src/main/java/ca/chronofit/chrono/circuit/CircuitTimerActivity.kt]: Why this

File: app/src/main/java/ca/chronofit/chrono/circuit/CircuitTimerActivity.kt [hunk 5]
```diff
@@ -2,14 +2,24 @@ package ca.chronofit.chrono.circuit
 
 import android.annotation.SuppressLint
 import android.app.Activity
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
 import android.media.AudioManager
 import android.media.ToneGenerator
+<<<<<<< HEAD:app/src/main/java/ca/chronofit/chrono/main/circuit/CircuitTimerActivity.kt
```
Comment (SVD6) [inline app/src/main/java/ca/chronofit/chrono/circuit/CircuitTimerActivity.kt]: Merge conflict?

**Review by SVD6 — CHANGES_REQUESTED** [review]

**Pull request merged**